supplementary_style_guide/master.adoc

```diff
@@ -164,6 +164,9 @@ include::glossary_terms_conventions/product_conventions/products_conventions_ove
 ==== Azure and .NET Core
 include::glossary_terms_conventions/product_conventions/azure-dotnet.adoc[leveloffset=+1]
 
+==== Partner and Hybrid Cloud Infrastructure
+include::glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc[leveoffset=+1]
```

Comment:
There is a typo here, leveoffset -> leveloffset (so the headings in the included file aren't being bumped to H5's properly)

Reply:
Thankyou!